crates/telemetry-subscribers/src/lib.rs

```diff
@@ -107,9 +117,8 @@ impl FilterHandle {
     }
 
     pub fn get(&self) -> Result<String, BoxError> {
-        self.0
-            .with_current(|filter| filter.to_string())
-            .map_err(Into::into)
+        let result = self.0.with_current(|filter| filter.to_string())?;
+        Ok(result)
```

Comment:
why this change?

Reply:
no reason, just seemed more clear in this way for me, reverted to `map_err` version